scripts/release/snapshot.diff

```diff
@@ -61,12 +61,26 @@
  # endregion replace-version dev
 
 === README.md ===
- # An external crate from a registry
--marker_lints = "<version>"
-+marker_lints = "0.1.0"
+ ```bash
+-curl -fsSL https://raw.githubusercontent.com/rust-marker/marker/v<version-major-minor>/scripts/release/install.sh | bash
++curl -fsSL https://raw.githubusercontent.com/rust-marker/marker/v0.1/scripts/release/install.sh | bash
  ```
-
-=== cargo-marker/README.md ===
+ ```ps1
+-curl.exe -fsSL https://raw.githubusercontent.com/rust-marker/marker/v<version-major-minor>/scripts/release/install.ps1 | powershell -command -
++curl.exe -fsSL https://raw.githubusercontent.com/rust-marker/marker/v0.1/scripts/release/install.ps1 | powershell -command -
+ ```
+ 
+-The provided scripts use a sliding git tag `v<version-major-minor>`, to allow for automatic patch version updates, however a fixed tag `v<version-major>.2.1` is also available.
++The provided scripts use a sliding git tag `v0.1`, to allow for automatic patch version updates, however a fixed tag `v0.1.0` is also available.
```

Comment:
The `v<version-major>.2.1` in this diff looks wrong. Shouldn't this be `v<version-major-minor>`? :thinking: 

The + line is correct again. This might just be a thing in the snapshot script.

Reply:
Ah, right, good catch. I though I fixed all the edge cases, but seems not. I'll fix the script